.gitignore

```diff
@@ -1 +1,3 @@
 /target
+config.yml
+.idea
```

Comment:
I'm generally against having editor-specific ignores in projects. 

Would be better to add this to a system gitignore: https://stackoverflow.com/a/7335487